src/averageAges.js

```diff
@@ -14,8 +14,19 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
+function averager(people) {
+  const agesOnly = people.map((person) => person.died - person.born);
+
+  return agesOnly.reduce((prev, curr) => prev + curr, 0) / agesOnly.length;
+}
+
+function calculateMenAverageAge(people, century = true) {
+  const men = people.filter(person => person.sex === 'm');
+  const centuryAges = men
+    .filter((person) =>
+      Math.ceil(person.died / 100) === century || century === true);
```

Comment:
So you've filtered array twice, but you can do it only once

Just use one filter method and using ternary operator inside it choose needed condition to return